var/spack/repos/builtin/packages/py-msal-extensions/package.py

```diff
@@ -16,9 +16,13 @@ class PyMsalExtensions(PythonPackage):
     homepage = "https://github.com/AzureAD/microsoft-authentication-library-for-python"
     pypi = "msal-extensions/msal-extensions-0.2.2.tar.gz"
 
+    version("1.0.0", sha256="c676aba56b0cce3783de1b5c5ecfe828db998167875126ca4b47dc6436451354")
     version("0.2.2", sha256="31414753c484679bb3b6c6401623eb4c3ccab630af215f2f78c1d5c4f8e1d1a9")
     version("0.1.3", sha256="59e171a9a4baacdbf001c66915efeaef372fb424421f1a4397115a3ddd6205dc")
 
+    # https://github.com/AzureAD/microsoft-authentication-extensions-for-python/blob/1.0.0/setup.py
     depends_on("py-setuptools", type="build")
     depends_on("py-msal@0.4.1:1", type=("build", "run"))
-    depends_on("py-portalocker@1.0:1", type=("build", "run"))
+    depends_on("py-portalocker@1", when="@:0", type=("build", "run"))
+    # This is the earliest version to work for Windows and non-Windows
+    depends_on("py-portalocker@1.6:1", when="@1:", type=("build", "run"))
```

Comment:
If you want to be technically correct this would look like:
```python
depends_on("py-portalocker@1.0:2", when="@1: platform=linux", type=("build", "run"))
depends_on("py-portalocker@1.0:2", when="@1: platform=darwin", type=("build", "run"))
depends_on("py-portalocker@1.0:2", when="@1: platform=cray", type=("build", "run"))
depends_on("py-portalocker@1.6:2", when="@1: platform=windows", type=("build", "run"))
```
but I don't really care either way

Reply:
I don't see a good reason to support older versions